src/wallet/wallet.cpp

```diff
@@ -146,6 +146,7 @@ std::shared_ptr<CWallet> LoadWallet(interfaces::Chain& chain, const WalletLocati
     }
     AddWallet(wallet);
     wallet->postInitProcess();
+
```

Comment:
in commit 596f91d20732a1714395e9f6e5d1e6c41f14a2e5:

Could remove empty hunk to avoid having to solve conflicts in a backport?